.github/workflows/downmerge-releases.yml

```
@@ -45,48 +51,73 @@ jobs:
           else
             BASE_PACKAGE_VERSION=$PACKAGE_VERSION
           fi
-          
-          CURRENT_MAJOR_MINOR_VERSION=$(echo $BASE_PACKAGE_VERSION | cut -d'.' -f1,2)
 
           # Iterate through branches and downmerge if they match the pattern
-          for branch in $(git for-each-ref --format='%(refname:short)' 'refs/remotes/origin/release/*' "refs/remotes/origin/${{ inputs.base-branch }}"); do
+          for branch in $(git for-each-ref --format='%(refname:short)' 'refs/remotes/origin/release-test/*' "refs/remotes/origin/${{ inputs.base-branch }}"); do
```

```suggestion
          for branch in $(git for-each-ref --format='%(refname:short)' 'refs/remotes/origin/release/*' "refs/remotes/origin/${{ inputs.base-branch }}"); do
```
revert after testing